<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="JefPc" failures="2" tests="12" name="juntos.Calculadora2Test" time="0.018" errors="0" timestamp="2023-06-01T21:36:40 BRT" skipped="0">
  <testcase classname="juntos.Calculadora2Test" name="testeMultiplicarDDT" time="0.001">
    <failure message="expected [-61.0] but found [-60.0]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [-61.0] but found [-60.0]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEquals(Assert.java:727)
at org.testng.Assert.assertEquals(Assert.java:774)
at juntos.Calculadora2Test.testeMultiplicarDDT(Calculadora2Test.java:116)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at juntos.Calculadora2Test.main(Calculadora2Test.java:23)
]]>
    </failure>
  </testcase> <!-- testeMultiplicarDDT -->
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeMultiplicarDDT" time="0.000"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeMultiplicar" time="0.000"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeSomar" time="0.001"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeMultiplicarDDT" time="0.000"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeDividirPorZero" time="0.001"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeSubtrair" time="0.000"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeMultiplicarDDT" time="0.007">
    <failure message="expected [1.0] but found [0.0]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [0.0]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEquals(Assert.java:727)
at org.testng.Assert.assertEquals(Assert.java:774)
at juntos.Calculadora2Test.testeMultiplicarDDT(Calculadora2Test.java:116)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at juntos.Calculadora2Test.main(Calculadora2Test.java:23)
]]>
    </failure>
  </testcase> <!-- testeMultiplicarDDT -->
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeMultiplicar2por0" time="0.000"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeDividir" time="0.008"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeMultiplicar22" time="0.000"/>
  <system-out/>
  <testcase classname="juntos.Calculadora2Test" name="testeMultiplicarDDT" time="0.000"/>
  <system-out/>
</testsuite> <!-- juntos.Calculadora2Test -->
